include/limits.h

```diff
@@ -68,6 +70,13 @@
 #  endif
 #endif
 
+/* Maximum length of any multibyte character in any locale.
+ * We define this value here since the gcc header does not define
+ * the correct value.
+ */
+
+#define MB_LEN_MAX            8
```

Comment:
why 8, not 1?

Reply:
Copy from lib_wctob.c it is fine to change to 1 since nuttx doesn't really support utf8 or other multiple bytes encoding. Do you prefer one?

Reply:
@xiaoxiang781216 

MB_LEN_MAX | Maximum number of bytes in a multibyte character, for any locale | 1 or greater*
* the actual value depends on the particular system and library implementation, but shall reflect the limits of these types in the target platform.

Reply:
Ok, I change to 1 base on the current implementation.